kotest-assertions/kotest-assertions-core/src/commonMain/kotlin/io/kotest/assertions/nondeterministic/eventually.kt

```diff
@@ -133,7 +138,7 @@ object EventuallyConfigurationDefaults {
 class EventuallyConfigurationBuilder {
 
    /**
-    * The total time that the [eventually] function can take to complete successfully.
+    * The total time that the [eventually] function can take to complete successfully. Must be greater than or equal to 0.
```

Comment:
maybe just "Must be non-negative"?

Reply:
omg....
I will fix formatting on evening! 👍

Reply:
I will fix this on evening!

Reply:
tbh I find "Must be greater than or equal to zero" easier to read, but "Must be non-negative" is fine too.